### ISSUE TITLE
NPM updates

### ISSUE THREAD

**Issue by danielkoch** [jantimon/iconfont-webpack-plugin#35]
Would be nice to update npm packages. Quick solution: Use `npm audit fix` for automatically update non-breaking updates.

*`npm audit` before:*
```
found 484 vulnerabilities (5 low, 479 moderate) in 17994 scanned packages
  run `npm audit fix` to fix 9 of them.
  475 vulnerabilities require semver-major dependency updates.
```
*`npm audit` after:*
```
fixed 9 of 484 vulnerabilities in 17994 scanned packages
  5 package updates for 475 vulns involved breaking changes
  (use `npm audit fix --force` to install breaking changes; or refer to `npm audit` for steps to fix these manually)
```
*`npm audit --force` report:*

```
+ commitizen@3.0.7
+ ava@1.2.1
+ webpack-cli@3.2.3
added 95 packages from 32 contributors, removed 369 packages, updated 70 packages and moved 27 packages in 20.781s
fixed 6 of 6 vulnerabilities in 17994 scanned packages
  3 package updates for 6 vulns involved breaking changes
  (installed due to `--force` option)
```
*`npm audit` after:*
```
found 0 vulnerabilities
 in 12786 scanned packages
```

**Comment by danielkoch** [comment]
I created a PR to fix the vulnerabilities: https://github.com/jantimon/iconfont-webpack-plugin/pull/36

**Comment by jantimon** [comment]
Released in 3.2
